clients/venice-thin-client/src/test/java/com/linkedin/venice/client/schema/RouterBackedSchemaReaderTest.java

```diff
@@ -255,20 +302,31 @@ public void testGetLatestValueSchemaWithSupersetSchema() throws Exception {
     try (SchemaReader schemaReader = new RouterBackedSchemaReader(
         () -> clientMock,
         Optional.empty(),
-        Optional.of(schema -> schema.toString().equals(valueSchemaStr1)),
+        Optional.of(schema -> schema.toString().equals(valueSchemaStr2)),
```

Comment:
Why do you have to change this? The intent of this test-case is to verify that preferred schema is returned even if it isn't the latest.